### DISCUSSION TITLE
Logo licence

### DISCUSSION THREAD

**Discussion by PhilipVinc** [jax-ml/jax#6007] [General]
Hi,

Over at [`mpi4jax`](https://github.com/PhilipVinc/mpi4jax) we were playing around with having a simple logo, while recalling Jax. I was thinking about something along the lines of what you see below (though probably with different colors).

Would that be ok, licence wise (I believe Apache 2 would allow it, but I'm not sure Jax logo is covered by the same licence) ?

<img width="773" alt="screen2" src="https://user-images.githubusercontent.com/2407108/110641578-7ca0b200-81b2-11eb-8ea3-3144b4216258.png">

**Comment by shoyer** [comment]
I would not assume that the logo is open source licensed. Something like the [TensorFlow Brand Guidelines](https://www.tensorflow.org/extras/tensorflow_brand_guidelines.pdf) probably should be assumed to apply. To be safe, I would suggest making something that plays _homage_ to the JAX logo without directly copying it.

(That said, in practice I would be surprised if somebody chases after you for adapting the logo in a community-driven project. We don't even sell JAX T-shirts!)
